README.md

```diff
@@ -45,6 +45,23 @@ The following `npm` scripts are defined for convenience:
 
 The TetraScience Agents produce a number of structured logs and metrics which provide insight into the execution of the agents. There is not, however, a productized solution for monitoring these metrics.  This repository is designed to be a starting point to leverage AWS CloudWatch to build a monitoring system (metrics and alarms) for your agents.
```

Comment:
suggestion: instead of "is designed to be a starting point" have something like "is a showcase of how to leverage AWS CloudWatch to ..."